utils/verify_intel_gaudi/roles/verify_intel_gaudi/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Initialise node accelerator status
+  ansible.builtin.set_fact:
+    node_has_accelerator: false
+
+- name: Check node has Gaudi2 or not
+  ansible.builtin.shell: |
+    set -o pipefail
+    lspci -n -d {{ gaudi2_pci_vendor_device_class }}
+  register: lspci_output
+  changed_when: false
+  failed_when: false
+  args:
+    executable: /bin/bash
+
+- name: Update node accelerator status
+  ansible.builtin.set_fact:
+    node_has_accelerator: true
+  when: lspci_output.stdout | length > 0
+
+- name: Include omnia_config.yml
+  block:
+    - ansible.builtin.include_tasks: include_omnia_config.yml
+  delegate_to: localhost
+
+- name: Verify hl-qual on Gaudi nodes
+  ansible.builtin.include_tasks: hlqual_validation.yml
+  when: node_has_accelerator
```

Comment:
When condition for verify_intel_gaudi_with_sanity_test_only can be added here by keeping these 2 tasks in block